addons/common/functions/fnc_runQTE.sqf

```diff
@@ -0,0 +1,143 @@
+#include "..\script_component.hpp"
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"
+/*
+ * Author: john681611
+ * Runs a Quick time Event.
+ *
+ * Arguments:
+ * 0: Object QTE is attached to <OBJECT>
+ * 1: Arguments, passed to condition, fail and finish <ARRAY>
+ * 2: On Finish: Code called or STRING raised as event. <CODE, STRING>
+ * 3: On Failure: Code called or STRING raised as event. <CODE, STRING>
+ * 4: QTE Seqence <ARRAY>
+ * 5: max interaction distance from attached object <NUMBER> (default: 10) 
+ * 6: timout <NUMBER> (default: 30) 
```

Comment:
I would like an `On Timeout` code block to this API

Reply:
Lets, keep this fairly simple for now and we can add more behaviour events